python/tvm/autotvm/task/relay_integration.py

```diff
@@ -0,0 +1,331 @@
+# pylint: disable=unused-variable,invalid-name
+"""
+Decorator and utilities for the integration with TOPI and Relay
+99.9% copy-paste of implementation by @MerryMercy
+
+"""
+import warnings
+import logging
+
+
+from ... import tensor, placeholder, create_schedule, target as _target
+
+from ..util import get_const_tuple
+from .task import create, register
+
+logger = logging.getLogger('autotvm')
+
+def serialize_args(args):
```

Comment:
We can remove this by adding custom json encoder and decoder in https://github.com/dmlc/tvm/blob/master/python/tvm/autotvm/record.py#L54

Reply:
The CI now breaks on `import nnvm` during the test. Is there a circular import now?